.github/renovate.json

```diff
@@ -19,36 +37,90 @@
         "^@types/eslint",
         "^@typescript-eslint/",
         "^eslint-",
-        "^eslint$"
+        "eslint$"
+      ]
+    },
+    {
+      "enabled": true,
+      "groupName": "GraphQL",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": [
+        "^@graphql/",
+        "^graphql-",
+        "graphql$"
       ]
     },
     {
       "enabled": true,
       "groupName": "Jest",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@types/jest$", "^jest", "^ts-jest$", "^jest-"]
+      "matchPackagePatterns": [
+        "^jest-",
+        "jest$"
+      ]
+    },
+    {
+      "enabled": true,
+      "groupName": "OpenTelemetry",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@opentelemetry/"]
     },
     {
       "enabled": true,
       "groupName": "Playwright",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^@playwright/", "^playwright$", "^playwright-"]
+      "matchPackagePatterns": [
+        "^@playwright/",
+        "^playwright-",
+        "playwright$"
+      ]
+    },
+    {
+      "enabled": true,
+      "groupName": "Prettier",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": [
+        "^prettier",
+        "prettier$"
+      ]
     },
     {
       "enabled": true,
       "groupName": "ProseMirror",
       "matchManagers": ["npm"],
-      "matchPackagePatterns": ["^prosemirror-"]
+      "matchPackagePatterns": [
+        "^prosemirror",
+        "prosemirror$"
+      ]
     },
     {
       "enabled": true,
       "groupName": "Sentry",
       "matchManagers": ["npm"],
       "matchPackagePatterns": ["^@sentry/"]
     },
+    {
+      "enabled": true,
+      "groupName": "Signia",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@signia/"]
+    },
+    {
+      "matchManagers": ["poetry"],
+      "commitMessageTopic": "Python package `{{depName}}`",
+      "additionalBranchPrefix": "py/"
+    },
+    {
+      "enabled": true,
+      "groupName": "Pydantic",
+      "matchManagers": ["poetry"],
+      "matchPackagePatterns": ["^pydantic"]
+    },
```

Comment:
@TimDiekmann Could I have combined these into one? I think possibly yes, right?

Reply:
Which one do you mean, the python related groups? I don't see how we could combine them as you add a group name for `Pydantic`. I don't know if we need that, though 🤷🏻‍♂️

Reply:
My brain is literally not working this morning. I'm going to take a nap.